src/model/client.ts

```diff
@@ -277,7 +376,10 @@ export class CommandoClient extends Client {
 
       this.registerCommand(instance)
     } catch (err) {
-      // swallow
+      // tslint:disable-next-line: no-unsafe-any
+      if (err.message.startsWith('Commando: ')) {
```

Comment:
Can you move the try catch in this method to only be around the `import` and line that creates an instance of the class (will need to change the variable to use `let`), and outside the try-catch, add an if statement to return from the method if the instance is undefined.

That would solve the need for this catch & rethrow statement.

This current solution has a couple issues...
1. Catch/Rethrowing should be avoided unless you rethrowing adds extra context to the exception
2. It relies heavily on the magic string `Commando: `. If an error message were to be refactored, this would get skipped over, and this logic would fail.

By refactoring the method as I described, it would allow the error to bubble up without needing any hacks to bypass the other error handling.